src/quo/components/navigation/page_nav/view.cljs

```diff
@@ -85,14 +87,29 @@
      :else
      nil)])
 
+(def header-height 155)
+(def page-nav-height 25)
+(def threshold (- header-height page-nav-height))
+
 (defn- title-center
-  [{:keys [centered? title center-opacity]}]
-  [rn/view {:style (style/center-content-container centered? center-opacity)}
-   [text/text
-    {:weight          :medium
-     :size            :paragraph-1
-     :number-of-lines 1}
-    title]])
+  [{:keys [centered? title center-opacity scroll-y]}]
+  (let [translate-animation (when scroll-y
+                              (header-worklet/header-content-position scroll-y
+                                                                      threshold
+                                                                      page-nav-height))
+        opacity-animation   (when scroll-y
+                              (header-worklet/header-content-opacity scroll-y threshold))]
```

Comment:
these worklets are using a hook inside (`useDerviedValue`) and we are conditionally wrapping them (`when`). This may lead to an exception since we must render the same amount of hooks across re-renders.

So I believe this `scroll-y` always exists otherwise the app is throwing an exception (and it's not doing it), so maybe we can remove them and probably not even mounting the component if there's no `scroll-y`

Reply:
yes the `scroll-y` either exist from the beginning or not, so it doesn't change during the runtime and doesn't throw any error.